### PR TITLE
feat: add multi sign support

### PR DESCRIPTION
**Pull request by fpChan**

### CODE REVIEW

File: offchain-modules/config.json.example [hunk 1]
```diff
@@ -57,7 +61,26 @@
...
+            "0xebf9befcd8396e88cab8fcb920ab149231658f4b"
+          ]
+      },
+      "keys": [
+       "0xd00c06bfd800d27397002dca6fb0993d5ba6399b4238b2f29ee9deb97593d2bc",
+       "0x63d86723e08f0f813a36ce6aa123bb2289d90680ae1e99d4de8cdb334553f24d"
+      ],
+      "hosts": [
+        "http://127.0.0.1:8090/force-bridge/sign-server/api/v1",
+        "http://127.0.0.1:8091/force-bridge/sign-server/api/v1"
```
Comment (huwenchao) [inline offchain-modules/config.json.example:81]: IMO, This config should be same for all xchain. Maybe move it into common?

File: offchain-modules/package.json [hunk 2]
```diff
@@ -121,9 +130,14 @@
     ]
   },
   "lint-staged": {
-    "packages/*/src/**/*.ts": [
+    "src/**/*.ts": [
```
Comment (huwenchao) [inline offchain-modules/package.json:133]: this part and the husky part seems like a merge error.
should use the version in main branch.

File: offchain-modules/config.json.example (hunk 1 above)
Comment (fpChan) [reply offchain-modules/config.json.example:81]: I consider doing this after the merge code

File: offchain-modules/packages/scripts/src/integration-test/eth.ts [hunk 3]
```diff
@@ -108,7 +109,13 @@ async function main() {
 
     // check sudt balance.
     const account = new Account(PRI_KEY);
-    const ownLockHash = ckb.utils.scriptToHash(<CKBComponents.Script>await account.getLockscript());
+    // const ownLockHash = ckb.utils.scriptToHash(<CKBComponents.Script>await account.getLockscript());
```
Comment (huwenchao) [inline offchain-modules/packages/scripts/src/integration-test/eth.ts:112]: delete this

Comment (huwenchao) [inline offchain-modules/packages/scripts/src/integration-test/eth.ts:112]:
```suggestion
```

File: offchain-modules/packages/scripts/src/integration-test/eth.ts [hunk 4]
```diff
@@ -136,7 +143,7 @@ async function main() {
     const burnAmount = ethers.utils.parseEther('0.01');
     if (!sendBurn) {
       const account = new Account(PRI_KEY);
-      const ownLockHash = ckb.utils.scriptToHash(<CKBComponents.Script>await account.getLockscript());
+      // const ownLockHash = ckb.utils.scriptToHash(<CKBComponents.Script>await account.getLockscript());
```
Comment (huwenchao) [inline offchain-modules/packages/scripts/src/integration-test/eth.ts:146]:
```suggestion
```

File: offchain-modules/packages/scripts/src/integration-test/tron.ts [hunk 5]
```diff
@@ -15,6 +16,7 @@ import nconf from 'nconf';
 import TronWeb from 'tronweb';
 import { createConnection } from 'typeorm';
 import { waitUntilCommitted } from './util';
+// import { multisigLockScript } from '@force-bridge/ckb/tx-helper/multisig/multisig_helper';
```
Comment (huwenchao) [inline offchain-modules/packages/scripts/src/integration-test/tron.ts:19]:
```suggestion
```

File: offchain-modules/packages/scripts/src/integration-test/tron.ts [hunk 6]
```diff
@@ -115,7 +117,13 @@ async function main() {
 
   const getBalance = async (assetName) => {
     const account = new Account(PRI_KEY);
-    const ownLockHash = ckb.utils.scriptToHash(<CKBComponents.Script>await account.getLockscript());
+    // const ownLockHash = ckb.utils.scriptToHash(<CKBComponents.Script>await account.getLockscript());
```
Comment (huwenchao) [inline offchain-modules/packages/scripts/src/integration-test/tron.ts:120]:
```suggestion
```

**Pull request merged**